tla-import/src/main/scala/at/forsyte/apalache/tla/imp/OpApplTranslator.scala

```diff
@@ -58,15 +61,68 @@ class OpApplTranslator(sourceStore: SourceStore, val context: Context, val recSt
 
   /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-  // call to a user-defined operator, either a LOCAL one, or not
-  private def translateUserOperator(node: OpApplNode) = {
-    if (node.getOperator.isLocal) {
-      translateLocalOperator(node)
+  // call to a user-defined operator (may come from the standard library), either a LOCAL one, or not
+  private def translateUserOperator(applyNode: OpApplNode) = {
+    if (!applyNode.getOperator.isLocal) {
+      translateNonLocalUserOperator(applyNode)
     } else {
-      translateNonLocalUserOperator(node)
+      // issue 295: https://github.com/informalsystems/apalache/issues/295
+      // LOCAL operators may originate from the standard library :face_palm
+      // For instance, if the user writes LOCAL INSTANCE Sequences
+      applyNode.getOperator match {
+        case opdef: OpDefNode =>
+          // translate as an application of a standard library operator
+          translateLocalLibraryOperatorOrValue(opdef, applyNode)
+            .getOrElse {
+              // or as an application of a local user-defined operator
+              translateLocalOperator(opdef, applyNode)
+            }
+
+        case _ =>
+          val msg = "Expected a LOCAL operator defined with OpDefNode, found: " + applyNode.getOperator.getClass
+          throw new SanyImporterException(msg)
+      }
     }
   }
 
+  // call an operator or a value (e.g., Sequences!Append or Integers!Int) from a standard module
+  // that is instantiated with LOCAL INSTANCE
+  private def translateLocalLibraryOperatorOrValue(opdef: OpDefNode, applyNode: OpApplNode): Option[TlaEx] = {
+    val source = opdef.getSource
+    if (source == null || source.getOriginallyDefinedInModuleNode == null) {
+      None
+    } else {
+      val operatorName = source.getName.toString
+      // the original module, in which the operator is defined
+      val modName = source.getOriginallyDefinedInModuleNode.getName.toString
+      StandardLibrary.libraryOperators.get((modName, operatorName)).map {
+            // a library operator
+        tlaOper =>
+          val exTran = ExprOrOpArgNodeTranslator(sourceStore, context, recStatus)
+          val args = applyNode.getArgs.toList.map { p => exTran.translate(p) }
+          OperEx(tlaOper, args: _*)
+      }.orElse {
+            // a library value
+        StandardLibrary.libraryValues.get((modName, operatorName)).map(ValEx)
+      }
+    }
+  }
+
+  // call a user-defined operator that is defined locally
+  private def translateLocalOperator(opdef: OpDefNode, node: OpApplNode): TlaEx = {
+    // Since we do not know the original location of the local operator,
+      // we can only re-define it with a LET-IN expression
```

Comment:
```suggestion
    // we can only re-define it with a LET-IN expression
```

I'll get scalafmt running on CI this week :)